scripts/linux/pkg/deb/assets/ravendb/debian/ravendb.postinst

```diff
@@ -4,9 +4,21 @@ set -e
 
 case "$1" in
     configure)
+        if [[ -n "$RAVEN_GROUP_ID" && $RAVEN_GROUP_ID -ge 1000 ]]; then
```

Comment:
Now I wonder... since it's passed in runtime and the package is being installed (and the user is created) during the build phase, we cannot really steer it what kind of UID/GID it's going to have, can we? Let's talk about this on Monday.